mesa/mesaPROTON_OC.py

```diff
@@ -194,88 +253,108 @@ def step(self):
                     agent.prisoner = False
         self.make_people_die()
         self.datacollector.collect(self)
-        # The datacollector assume that agents have a step method
         self.schedule.step()
         self.ticks += 1
 
-    def run(self, n_agents, ticks):
+
+    def run(self, n_agents: int, ticks: int, verbose: bool = False) -> None:
+        """
+        Run the model setup with @n_agents agents and execute @ticks tick.
+
+        :param n_agents: int, number of agents
+        :param ticks:int, number of agents
+        :param verbose: bool, extended console printing
+        :return:
+        """
+        self.verbose = verbose
         self.setup(n_agents)
-        for i in tqdm(range(ticks)):
+        pbar = tqdm(np.arange(0, ticks))
+        for tick in pbar:
             self.step()
+            pbar.set_description("tick: %s" % tick)
 
-    def fix_unemployment(self, correction):
-        available = [x for x in self.schedule.agents if x.age > 16 and x.age < 65 and x.my_school == None]
-        unemployed = [x for x in available if x.job_level == 1]
-        occupied = [x for x in available if x.job_level > 1]
-        notlooking = [x for x in available if x.job_level == 0]
+
+    def fix_unemployment(self, correction: Union[float, int, str]) -> None:
+        """
+        This function strictly depends on the Proton_OC.unemployment_multiplier parameter.
```

Comment:
always clean language in comments so that only the essential remains - no "strictly".

Furthermore: when you say "This function strictly depends on the Proton_OC.unemployment_multiplier parameter." you're not describing the function but anticipating usage. The function knows nothing about that parameter, it's passed on to her.


Here, the comment exposes a weakness - "